Substrate.NET.Wallet/Extensions/BytesExtension.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Schnorrkel.Merlin;
+using System;
+using System.Linq;
+using System.Security.Cryptography;
+
+namespace Substrate.NET.Wallet.Extensions
+{
+    public static class BytesExtension
+    {
+        /// <summary>
+        /// Load a byte array with random bytes
+        /// </summary>
+        /// <param name="data"></param>
+        /// <returns></returns>
+        public static byte[] Populate(this byte[] data)
+        {
+            var randomGenerator = RandomNumberGenerator.Create();
```

Comment:
In the Populate method, you're creating a new instance of RandomNumberGenerator each time you call the method. It would be more efficient to reuse the instance.

```csharp
     private static readonly RandomNumberGenerator RandomGenerator = RandomNumberGenerator.Create();

    /// <summary>
    /// Load a byte array with random bytes
    /// </summary>
    /// <param name="data"></param>
    /// <returns></returns>
    public static byte[] Populate(this byte[] data)
    {
        RandomGenerator.GetBytes(data);
        return data;
    }
```